build.gradle

```diff
@@ -15,15 +15,14 @@
 
 buildscript {
     ext {
-        es_version = System.getProperty("es.version", "7.4.2")
-        kotlin_version = System.getProperty("kotlin.version", "1.3.31")
+        es_version = System.getProperty("es.version", "7.6.1")
+        kotlin_version = System.getProperty("kotlin.version", "1.3.61")
     }
 
     repositories {
+        mavenLocal()
```

Comment:
Why was this moved above other repositories?

Reply:
Though it helps gradle to find local publish